extensions/panache/hibernate-orm-panache-rest/runtime/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-hibernate-orm-panache-rest-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-hibernate-orm-panache-rest</artifactId>
+    <name>Quarkus - Hibernate ORM with Panache REST Controllers - Runtime</name>
+
```

Comment:
We will need a description here. It will be the one displayed on code.quarkus.io. You can have a look there to have examples.